ui/packages/shared/profile/src/MetricsGraph/UtilizationMetrics/index.tsx

```diff
@@ -83,13 +84,26 @@ function transformToSeries(data: MetricSeries[]): Series[] {
   }));
 }
 
+const getYAxisUnit = (name: string): string => {
+  switch (name) {
+    case 'GPU Utilization':
+      return 'percent';
+    case 'GPU Memory Utilization':
+      return 'percent';
+    case 'GPU Power':
+      return 'watts';
+    default:
+      return 'percent';
+  }
```

Comment:
I'm not sure where the name variable comes from, but it feels like it should be based on the metric name and not on the axis label. Otherwise looks good to me.